demos/signature/EVP_Signature_demo.c

```diff
@@ -0,0 +1,238 @@
+/*-
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* 
+ * An example that uses the EVP_MD*, EVP_DigestSign* and EVP_DigestVerify* 
+ * methods to calculate and verify a signature of two static buffers.
+ * Another way of generating a signature is to use EVP_PKEY_sign(), it uses
```

Comment:
There should be a demo of this alternative.  Either add such a demo or remove this part of the comment -- it's not particularly helpful.

Reply:
Pauli - that was my suggestion - i.e. there are multiple ways of doing this..

Reply:
I don't like the dangling reference: here is a way that uses three calls.  Oh, there's a way that only uses one but I'm not telling.

Reply:
Done in latest upload, thanks for your comments.